lib/debugger/formatTable.luau

```diff
@@ -1,3 +1,39 @@
+local stringColor = "#dcdcaa"
+local numberColor = "#b0c4de"
+local keywordColor = "#c586c0"
+
+function colorize(value, color)
```

Comment:
Due to the requested change in colorize many, you have to escape if color is nil. 

```luau
function colorize(value, color)
    if not color then 
        return 
    end
end